tsconfig.json

```diff
@@ -59,6 +59,7 @@
     "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
 
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "skipLibCheck": true
```

Comment:
You should remove this option.

Reply:
If remove this, some packages will cause errors.